src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -718,6 +718,13 @@
     <Compile Include="$(MSBuildThisFileDirectory)Internal\IO\File.Windows.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeFileHandle.Windows.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeFindHandle.Windows.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Collections\Concurrent\CDSCollectionETWBCLProvider.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Collections\Concurrent\ConcurrentBag.cs" />
```

Comment:
Why are we adding ConcurrentBag to corelib?  What in corelib needs it?

Reply:
I wasn't sure if it's best to move ConcurrentBag along with the rest. corelib does not need it as you pointed out.

This PR and the corefx PR:  dotnet/corefx#29356 together work fine as long as we wanted to move all of them over to coreclr.
Now that I updated the PRs, I will work on your comment below and try to tweak them to only move over ConcurrentQueue instead.